fobi_custom/plugins/form_handlers/collect_data/fobi_form_handlers.py

```diff
@@ -85,4 +78,44 @@ def plugin_data_repr(self):
     return self.data.__dict__
 
 
+def get_element_info(self, form_element):
```

Comment:
Any reason these functions can't be methods on the `CollectDataPlugin` class?

Reply:
They can!